app/controllers/locker_applications_controller.rb

```diff
@@ -98,8 +100,8 @@ def lookup_user_from_params(locker_params)
     uid = locker_params.delete(:user_uid)
     user = if current_user.uid == uid
              current_user
-           else
-             User.find_by(uid: uid)
+           elsif current_user.admin?
```

Comment:
Could you please add a test for if a non-admin attempts to pass in somebody else's uid?

Reply:
Done!